cli/lib/kontena/scripts/completer

```diff
@@ -3,238 +3,7 @@
 
 # resolve bin path, ignoring symlinks
 require 'pathname'
-require 'yaml'
 bin_file = Pathname.new(__FILE__).realpath
+$:.unshift File.expand_path('../../..', bin_file)
```

Comment:
I think it was broken previously but still worked because the correct libpath came from the fact that kontena-cli gem was installed to system gems.

```
irb(main):001:0> File.expand_path('../../lib', '/Users/kimmo/Projects/kontena/cli/lib/kontena/scripts/completer')
=> "/Users/kimmo/Projects/kontena/cli/lib/kontena/lib"
```